en/sql/query/show.rst

```diff
@@ -384,7 +386,7 @@ The LIKE condition without the WHERE condition is applied on the first column. T
 
 .. code-block:: sql
 
-    SHOW FULL TIMEZONES;
+    SHOW [ FULL ] TIMEZONES [ LIKE 'pattern' ];
```

Comment:
I think previous 'SHOW FULL TIMEZONES' is right.

Reply:
Thanks, I'll change it.
I made a mistake thinking it was a grammar explanation.